control_plane/src/storage_controller.rs

```diff
@@ -346,7 +346,14 @@ impl StorageController {
             let pg_log_path = pg_data_path.join("postgres.log");
 
             if !tokio::fs::try_exists(&pg_data_path).await? {
-                let initdb_args = ["-D", pg_data_path.as_ref(), "--username", &username()];
+                let initdb_args = [
+                    "-D",
+                    pg_data_path.as_ref(),
+                    "--username",
+                    &username(),
+                    "--no-sync",
```

Comment:
eventually we will sync though, right?

Reply:
nope, in tests we don't try to be crash-safe

Reply:
yeah, we'd want to recreate the db anyways from scratch.